src/customFileParser/unpackGeometry/main.ts

```diff
@@ -38,23 +39,23 @@ function findOrCreateDestinationGroup(
   if (destinationGroup) {
     return destinationGroup;
   } else {
-    if (originalSector.children.length === 0) {
-      const capacity = renderedPrimitiveInfo.count * numOfGeometries;
-      return createGroup(originalSector, renderedPrimitiveInfo.name, capacity);
-    } else {
-      return createGroup(originalSector, renderedPrimitiveInfo.name, 1000);
-    }
+    // @ts-ignore
+    const capacity = Math.min(window.maxCapacity,
```

Comment:
`window.` should probably not be used :D

Reply:
Whoops, that got into the wrong branch. (I was using window extensively for my performance tests)